version/version.go

```diff
@@ -63,33 +102,48 @@ func (v Version) Format(format string) (string, error) {
 
 	var buf buffer
 
+	major := v.Major
```

Comment:
nit: Maybe use a `var`-block instead
```go
	var (
		buf   buffer
		major = v.Major
		minor = v.Minor
		patch = v.Patch
	)
```

Reply:
I did not know about `var`-blocks ;) Looks neat indeed.